corehq/apps/domain/forms.py

```diff
@@ -951,9 +951,11 @@ def __init__(self, *args, **kwargs):
         if not SECURE_SESSION_TIMEOUT.enabled(domain):
             excluded_fields.append('secure_sessions_timeout')
 
-        # PrependedText ensures the label is to the left of the checkbox, and the help text beneath.
-        # Feels like there should be a better way to apply these styles, as we aren't pre-pending anything
-        fields = [twbscrispy.PrependedText(field_name, '')
+        for field in self.fields.values():
+            if not isinstance(field.widget, BootstrapCheckboxInput):
```

Comment:
In case someone adds a field with a customized `BootstrapCheckboxInput` that has an inline label or something along those lines. This code would blow it away without the if statement, and it could be difficult to debug.